### PR TITLE
Update doc and typings for res.json()

### PR DESCRIPTION
**Pull request by fralonra**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,6 +140,7 @@ Injects a fake request into an HTTP server.
     - `body` - alias for payload.
     - `rawPayload` - the raw payload as a Buffer.
     - `trailers` - an object containing the response trailers.
+    - `json` - a function parses the JSON format payload.
```
Comment (RafaelGSS) [inline README.md]:
```suggestion
    - `json` - a function that parses string to json.
```

File: README.md [hunk 2]
```diff
@@ -140,6 +140,7 @@ Injects a fake request into an HTTP server.
     - `body` - alias for payload.
     - `rawPayload` - the raw payload as a Buffer.
     - `trailers` - an object containing the response trailers.
+    - `json` - a function that parses string to json.
```
Comment (Eomm) [inline README.md]: I would improve the description:
`json()` - parse the application/json response payload and returns an object. Throws if the content type is different

**Comment by mcollina** [comment]
Would you mind to add a test for the new type?

Il giorno mar 15 ott 2019 alle 07:30 Manuel Spigolon <
notifications@github.com> ha scritto:

**Comment by fralonra** [comment]
@mcollina 
I'm sorry, but I don't understand how to `add a test for the new type`.

**Comment by mcollina** [comment]
Just add a call to `json()` in https://github.com/fastify/light-my-request/blob/e9df4c47749a2b2a794d20e77d5a047d690290a3/test/index.test-d.ts#L17.

**Comment by fralonra** [comment]
@mcollina should I use `expectType` to test if the type of `res.json` is `Function`?

**Comment by mcollina** [comment]
That'd be nice.

This needs a rebase.

**Comment by fralonra** [comment]
@mcollina  Rebase and then force push? By me?

**Comment by mcollina** [comment]
I cannot push to your branch. You can also merge master.

**Comment by fralonra** [comment]
Done. BTW, can you rebase and merge when merging to master? @mcollina

**Pull request merged**